lib/takeaway.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'order'
+require_relative 'menu'
+require_relative 'text_confirmation'
+
+class Takeaway
+
+  def initialize(menu = Menu.new, order = Order.new, text_confirmation = TextConfirmation.new)
+    @menu = menu
+    @order = order
+    @text_confirmation = text_confirmation
+  end
+
+  def read_menu
+    @menu.food_options
+  end
+
+  def make_selection(food_order, quantity = 1)
```

Comment:
It's great that Takeaway has methods like `order_summary`, `make_selection`, `read_menu`. This makes it like a user interface class for the user to interact with.  But it's still very focused because it delegates the actual work to other classes.